spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/callback/SpringSecurityPasswordValidationCallbackHandlerTest.java

```diff
@@ -48,6 +50,15 @@
 	private WSPasswordCallback passwordCallback;
 
 	private UserDetails user;
+	
+	/**
+	 * add a tearDown method at the beginning, 
+	 * in case {@code SecurityContextHolder} not cleaned before this class
+	 */
+	@BeforeAll
+	public static void beforeClassTearDown() { 
+		SecurityContextHolder.clearContext();
+	}
```

Comment:
Why not simply put this statement inside the existing `setUp()` method?

Reply:
Thanks for your suggestion, I edited the code and moved the `clearContext ()` method into setup to make it more simple than before.